plugins/module_utils/network/ios/rm_templates/bgp_address_family.py

```diff
@@ -80,7 +80,7 @@ def __init__(self, lines=None, module=None):
                 r"""
                 \s\saggregate-address
                 (\s(?P<address>\S+))?
-                (\s(?P<netmask>\S+))?
+                (\s(?P<netmask>(?!as-set|summary-only|as-confed-set|advertise-map|attribute-map|suppress-map)\d+\.\d+\.\d+\.\d+))?
```

Comment:
i am not sure if we need to match pattern here or we can just change the order of the sections to make it just work?